project_5.html

```diff
@@ -0,0 +1,697 @@
+			<li class="list-group-item">The LED must be brightly lit when turned ON.</li>
+			<li class="list-group-item">The LED must be completely dark when turned OFF.</li>
+			<li class="list-group-item">The circuit must be built on a breadboard.</li>
+		</ul>
+		<h3 class="mb-3" style="display: inline;" id="pj-heading-internal-pull-up">Internal Pull-Up Resistor Circuit</h3>
+		<ul class="mb-5 list-group">
+			<li class="list-group-item">Using the Arduino board, you must build a digital LED circuit which is activated by a pushbutton connected to an internal pull-up resistor. </li>
+			<li class="list-group-item">When the pushbutton is pressed, the built-in LED (pin D13) on the Arduino board should turn ON.</li>
+			<li class="list-group-item">The build-in LED must turn OFF with the pushbutton is released.</li>
+			<li class="list-group-item">The circuit must NOT use an external pull–up resistor.</li>
+			<li class="list-group-item">The LED must be brightly lit when turned ON.</li>
+			<li class="list-group-item">The LED must be completely dark when turned OFF.</li>
+			<li class="list-group-item">The circuit must be built on a breadboard.</li>
+		</ul>
+		<h3 class="mb-3" style="display: inline;" id="pj-heading-ipoduino">iPoduino</h3>
+		<ul class="mb-5 list-group">
+			<li class="list-group-item">You must build a MP3 Player with speaker sound using the DFPlayerMini module.</li>
+			<li class="list-group-item">The player must be pre-loaded with 3 songs.</li>
+			<li class="list-group-item">Each song must be at least 30 seconds in duration.</li>
+			<li class="list-group-item">The player must have two buttons.</li>
+			<li class="list-group-item">
+				<p>
+					One of the player’s buttons must toggle the music between “Play” mode and “Pause” mode.
+				<ul>
+					<li>In “Play” mode, the player must playback a song. It should resume the song that was previously paused if the “Skip/Next Song” button was not pressed.</li>
+					<li>In “Pause” mode, no song should be played.</li>
+				</ul>
+				</p>
+			</li>
+			<li class="list-group-item">
+				<p>
+					The second button should be the “Skip/Next Song” button, which interrupts the current song and plays a new song.
+				<ul>
+					<li>Each press of this button must cycle through all songs loaded on the MicroSD card.</li>
+					<li>If the player was in “Pause” mode when the “Skip/Next Song” button is pressed, the player must immediately enter “Play” mode.</li>
+				</ul>
+				</p>
+			</li>
+			<li class="list-group-item">The speaker must be silent when no song is played. No static or white noise.</li>
+			<li class="list-group-item">The speaker must clearly output music when a song is played.</li>
+			<li class="list-group-item">The circuit must be built on a breadboard.</li>
+
+		</ul>
+
+
+
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-parts">Parts</h2>
+
+			<!-- ======= Part Table ======= -->
+			<table class="table pj-part-table mb-5">
```

Comment:
Please fix the indentation in this section. The indentation distance between each nested tag is too big.